### ISSUE TITLE
Rehash the long-gone Carpotoxin exclusivity chemistry meta

### ISSUE THREAD

**Issue by 9600bauds** [d3athrow/vgstation13#8643]
Rezadone being made from Carpotoxin was a refreshingly novel idea of a useful chemical only being rarely available through random events or treasure hunts. That is, before all the other ways of getting Carpotoxin were added, now carps are the least often used method of getting Carpotoxin.
I think this is a waste, and it's a shame that the concept is not used elsewhere.
My suggestion is to replace Carpotoxin in the Zombie Powder and Rezadone recipes with some other chemical, make Groan's Soda give that chemical instead, and make Carpotoxin the ingredient for some other useful chemical. Maybe Peridaxon? Maybe something borrowed from Goonchem?
The bottom line is that we desperately need more inventive chemistry that uses anything you can't just make in a chemdispenser.

**Comment by Imagebard** [comment]
>Groan's Soda has Carpotoxin

Better idea, lets remove the Groan's Soda drink out of the Dispenser.
Goofball changes to the codebase weren't accepted by the playerbase anyways, so it should be of little concern.

**Comment by 9600bauds** [comment]
You can still get it from any other source of Zombie Powder. Which is not even necessarily a bad thing, it's just a shame that it stepped over carps as a source of rare chemicals. At this point it would be better to just leave Groan's Soda and Zombie Powder to another made-up chemical, and make Carpotoxin do something else.

**Comment by Imagebard** [comment]
Well, if you want to code an entirely new feature just to accommodate Carpotoxin, I guess It can't be helped.

**Comment by Exxion** [comment]
Carpotoxin does currently have a couple of uses other than rezadone, and with carp taming, carp pheromones are actually useful

**Comment by AutismSpider** [comment]
Carp pheromones are only safe to use vs baby s full size carp will fuck you in the ass if you don't have pheromones in you 1st

**Comment by Exxion** [comment]
Yes, but you can tame the baby carp without dying, then

**Comment by Probe1** [comment]
There is nothing worthwhile about either Rezadone or Peridaxon and it can be used as a text book example of dead end code that sounded good on paper.  

It is too niche and rare for most players to know about it so in the off chance you do get some, you have to have a tutorial class in the middle of the wild ride about what a chem does.

Compounded on this is rezadone and to a lesser extent peridaxons underwhelming nature.  You got 50 units of rezadone?  That's great.  That's enough to remove the clone damage from uh.. maybe one person or make cryo 5% faster.

Peridaxon works better since organ damage is usually a very low number but clone damage is the opposite case.

_None of this addresses the issue raised._  But let's get real if anythings going to be changed - it's shit as is.  They're balanced like they're available from a chem dispenser but incredibly tedious to get.

**Comment by Iamgoofball** [comment]
@Imagebard i know that the goonchem touched you in a bad place and you seek the need to eliminate anything related to my name with no care or concern, but groans soda is needed for ghettochem and is in fact a key component in the ghettochem meta

**Comment by 9600bauds** [comment]
> You got 50 units of rezadone? That's great. That's enough to remove the clone damage from uh.. maybe one person or make cryo 5% faster.

I get 75u Rezadone every shift, it's 11 pills of making Cryo 25-50% faster. It's pretty damn good. Peridaxon is a fucking godsend but it's practically unattainable.

@Iamgoofball I have never, ever used groan's for chettochem because it's shit and it's extremely expensive

**Comment by Probe1** [comment]
That's about the level I get and the results are marginal compared to less time better spent making a robust cryo mix.

**Comment by Iamgoofball** [comment]
@9600bauds groans and discount dan's shit was supposed to be cheap as fuck, did someone bump up the prices?

the groans banned premixed shit can be deconstructed with the toilet chem deconstructor for some good shit

**Comment by 9600bauds** [comment]
Probe I have dedicated downright autismal amounts of time and effort to researching Cryo and Cryomixes and let me tell you that those "robust" mixes of 10 separate chemicals are a trap, are a waste of time, and are downright detrimental in nearly every case.
Besides, clone damage is only healed by 3 chemicals and all mixes need to have 2 of those.

**Comment by Probe1** [comment]
Do a side by side and shut me up.  We've both been doing this a long time but I've been at chemistry for several years and rezadone is a niche interest for top end chemists that want to be the best.  It is not a practical chem and fucking with it to make it anything except more commonly available is not a good idea.

**Comment by 9600bauds** [comment]
Idunno when was the last time you've played Chemistry but now we have trail mixes that make getting Rezadone extremely easy, I think it's pretty good as is

**Comment by AutismSpider** [comment]
@Iamgoofball yeah some one jack up the cost for the groans sodas (how i get my fish juice/poly acid)  the food is cheap though

**Comment by Imagebard** [comment]
@Iamgoofball Why should ghetto chemists have access to Carpotoxin, again?

**Comment by 9600bauds** [comment]
I think it's a good gameplay element. I can get an optional chem by doing some optional out-of-dispenser errands. Just the same though, there should be more of that.

**Comment by AutismSpider** [comment]
@Imagebard got hack the vend 1st then play RNG if the soda with carpotxin comes out any way at 100$ a pop

**Comment by Probe1** [comment]
Up the quality of rezadone then to bring it in line with peridaxons usefulness.  That's all I'm saying.

Rezadones exactly as useful as hacking a groans and leaving a monkey in front of it to catch the ejected bottles.  If it were any more work to get then the worth would not meet the work.

**Comment by 9600bauds** [comment]
Probe you can get rezadone by hacking a MediVend and scanning the trail mix bottles for zombie powder pills, it's easy as fuck and it's the only reason I get it every shift

**Comment by Probe1** [comment]
Well that's new.

Do something better and get rid of this static contraband funk we've gotten ourselves into and replace it with a random list.  Maybe it's trail mix or maybe it's O- or maybe it's an incision manager.

**Comment by 9600bauds** [comment]
There also is O-

**Comment by Probe1** [comment]
Do eet

**Comment by 9600bauds** [comment]
I just want to actually get out of Chemistry to make at least ONE good chemical holy shit
DD is kind of there but it's not even that useful

**Comment by Probe1** [comment]
I totally agree.  I did not know someone added it to med vends.  I would personally recommend using a randomized list for contraband so it is not _always_ available right there in Medbay.  It opens up an avenue for more beneficial items to be added since it won't be a fixed, predictable list, and it sends chemists out of their caves most of the time to get carpotoxin.

Having a gamble increases the feeling of reward and improves enjoyment.  

Advanced first aid kits, or advanced organs, or trail mix, or O- blood, or or or.

**Comment by Shadowmech88** [comment]
We need more unique reagents, stuff that can only be obtained from other departments and/or space exploration.
By extension, we'll need more chems that can use such reagents in their recipes.

What good Goon chems would fit in well?

**Comment by 9600bauds** [comment]
Ideally I wanted something that would be useful for Medbay, because otherwise you get lol hairgrownium!!

**Comment by Probe1** [comment]
Hairgrownium sounds like a fun drink effect.

**Comment by Shadowmech88** [comment]
The problem with "useful for Medbay" is that we've already got chems for healing literally everything. Sure, we could make better chems, but most of the ones we have already are good enough that no one would go the extra mile to get the "super burn heal chem" when 50u dermaline will heal any burn victim just fine and is much easier to make.

**Comment by Probe1** [comment]
#BRINGBACKOVERDOSES
